chromium: replace `systemd` with `systemdLibs`

drupol opened the pull request:
Hello,

I was not able to test this PR as I couldn't build this on my laptop.

This PR is not an emergency, so, take your time, no hurry on my side.

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

drupol committed:
chromium: replace `systemd` with `systemdLibs`

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/prs-already-reviewed/2617/2552

emilylange commented:
The correct maintainers were pinged and this is on our (@networkException and I) radar.
We take strong ownership over the `chromium` derivation and I personally find posting this PR in [_PRs already reviewed_](https://discourse.nixos.org/t/prs-already-reviewed/2617/2552) less than a week after it was marked as ready for review and knowing we were pinged pushy and unasked for. I have more important stuff to do [in nixpkgs] and this will have to wait for a bit longer.

Why is the PR template empty?
Why didn't you mention you haven't tested nor even built this in the beginning and instead had to be asked?
Why are there unrelated style changes in this tiny diff?

You could have just opened an issue for this if you aren't equipped to build and test this yourself.
I am not making fun of you for not being able to build this. We had `chromium` PRs by authors that did not have the compute either the in the past.

I take issue that you aren't communicating like you should, appearing pushy to get this merged and including unrelated style changes. You should know better as committer.

Like mentioned above in the first few sentences, this will have to wait. Expect me to come back to you in 1-2 weeks the earliest.

emilylange committed:
chromium: remove redundant `sed -i` for `libudev.so`

This sed is a no-op since [CL 5738848], which reached chromium stable
as part of M129 back in September 2024.

Electron 32, which was based on M128, reached EOL in March 2025.

As such, this can be safely removed without the need of a version
conditional.

[CL 5738848]: https://chromium-review.googlesource.com/c/chromium/src/+/5738848

emilylange reviewed (commented):
Thank you for your patience and for dropping the unrelated changes.

I took the liberty to force-push b744bf89d3463f865d0ea3f096a6a907b70c8d38 and rebase your commit on top of it, as you ticked the little "Maintainers are allowed to edit this pull request." checkbox.

I will have to build all of electron-source and chromium now, followed by some manual testing. Expect to hear back from me tomorrow or the day after tomorrow.

drupol commented:
Waiting is totally fine, no worries, I was just cleaning my open PRs this morning and saw that this one was not yet addressed.

Thanks for taking care of it, much appreciated :)

emilylange reviewed (approved):
The `electron_39` failure (and its dependent `netron`) is unrelated to this change here. See <https://github.com/NixOS/nixpkgs/pull/459455#issuecomment-3518742383>.

Besides that, manual testing of `chromium`, `ungoogled-chromium`, `electron_37` and `electron_38` did not reveal any obvious breakage, which is expected and good.

And of course, the VM tests for `chromium` and `ungoogled-chromium` also pass.

Merging.

---

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 441286 --skip-package ddm --skip-package sitespeed-io`
Commit: `551a4e6ad55c9b7ecbf5d3d3ac621d32eea17be2`

---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>electron_39</li>
    <li>netron</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 139 packages built:</summary>
  <ul>
    <li>affine</li>
    <li>affine-bin</li>
    <li>antares</li>
    <li>antimatter-dimensions</li>
    <li>anytype</li>
    <li>appium-inspector</li>
    <li>authentik</li>
    <li>aws-azure-login</li>
    <li>bilibili</li>
    <li>bitwarden-desktop</li>
    <li>bitwarden-directory-connector</li>
    <li>blockbench</li>
    <li>breitbandmessung</li>
    <li>bruno</li>
    <li>bs-manager</li>
    <li>camunda-modeler</li>
    <li>caprine</li>
    <li>chatd</li>
    <li>cherry-studio</li>
    <li>chromedriver</li>
    <li>chromium</li>
    <li>chromium.sandbox</li>
    <li>decktape</li>
    <li>deltachat-desktop</li>
    <li>drawio</li>
    <li>drawio-headless</li>
    <li>e-search</li>
    <li>electron (electron_38)</li>
    <li>electron_37</li>
    <li>element-desktop</li>
    <li>ente-desktop</li>
    <li>equibop</li>
    <li>fast-cli (nodePackages.fast-cli)</li>
    <li>fastmail-desktop</li>
    <li>fcast-receiver</li>
    <li>figma-linux</li>
    <li>flaci</li>
    <li>flaresolverr</li>
    <li>folo</li>
    <li>freetube</li>
    <li>gdlauncher-carbon</li>
    <li>gitify</li>
    <li>goofcord</li>
    <li>google-chat-linux</li>
    <li>google-lighthouse</li>
    <li>gotenberg</li>
    <li>gotenberg.hyphen</li>
    <li>grafana-kiosk</li>
    <li>gridtracker2</li>
    <li>headset</li>
    <li>heroic</li>
    <li>heroic-unwrapped</li>
    <li>html2pdf</li>
    <li>httptoolkit</li>
    <li>itch</li>
    <li>ivpn-ui</li>
    <li>jitsi-meet-electron</li>
    <li>joplin-desktop</li>
    <li>kando</li>
    <li>koodo-reader</li>
    <li>kopia-ui</li>
    <li>kuro</li>
    <li>legcord</li>
    <li>logseq</li>
    <li>marktext</li>
    <li>mattermost-desktop</li>
    <li>mermaid-cli</li>
    <li>mermaid-filter</li>
    <li>morgen</li>
    <li>mqtt-explorer</li>
    <li>nix-tour</li>
    <li>nodePackages_latest.fast-cli</li>
    <li>obsidian</li>
    <li>openfortivpn-webview</li>
    <li>pandoc-drawio-filter</li>
    <li>pandoc-drawio-filter.dist</li>
    <li>penpot-desktop</li>
    <li>percollate</li>
    <li>pocket-casts</li>
    <li>podman-desktop</li>
    <li>pritunl-client</li>
    <li>proton-pass</li>
    <li>protonmail-desktop</li>
    <li>puppeteer-cli</li>
    <li>python312Packages.mkdocs-drawio-exporter</li>
    <li>python312Packages.mkdocs-drawio-exporter.dist</li>
    <li>python313Packages.mkdocs-drawio-exporter</li>
    <li>python313Packages.mkdocs-drawio-exporter.dist</li>
    <li>r2modman</li>
    <li>redisinsight</li>
    <li>revolt-desktop</li>
    <li>ride</li>
    <li>rstudio</li>
    <li>rstudioWrapper</li>
    <li>selendroid</li>
    <li>selenium-server-standalone</li>
    <li>session-desktop</li>
    <li>sharedown</li>
    <li>shogihome</li>
    <li>sieve-editor-gui</li>
    <li>signal-desktop</li>
    <li>single-file-cli</li>
    <li>siyuan</li>
    <li>slacky</li>
    <li>splayer</li>
    <li>standardnotes</li>
    <li>stretchly</li>
    <li>super-productivity</li>
    <li>teams-for-linux</li>
    <li>terra-station</li>
    <li>tetrio-desktop</li>
    <li>thedesk</li>
    <li>threema-desktop</li>
    <li>trilium-desktop</li>
    <li>uhk-agent</li>
    <li>uhk-udev-rules</li>
    <li>uivonim</li>
    <li>undetected-chromedriver</li>
    <li>ungoogled-chromium</li>
    <li>ungoogled-chromium.sandbox</li>
    <li>unofficial-homestuck-collection</li>
    <li>vacuum-tube</li>
    <li>vesktop</li>
    <li>vhs</li>
    <li>voicevox</li>
    <li>vrcx</li>
    <li>vscode-extensions.yzane.markdown-pdf</li>
    <li>wayback</li>
    <li>webtorrent_desktop</li>
    <li>whalebird</li>
    <li>whatsapp-electron</li>
    <li>winboat</li>
    <li>yandex-music</li>
    <li>youtube-music</li>
    <li>ytdownloader</li>
    <li>ytmdesktop</li>
    <li>zap-chip-gui</li>
    <li>zepp-simulator</li>
    <li>zulip</li>
  </ul>
</details>
